packages/NodeTypeResolver/Node/AttributeKey.php

```diff
@@ -77,6 +77,7 @@ final class AttributeKey
     public const PARENT_NODE = 'parent';
 
     /**
+     * @api
```

Comment:
even if the keys exist the attributes will no longer exist right?
do we need some kind of "bc preserving" mode which registers the `NodeConnectingVisitor` instead of `ParentConnectingVisitor `?

in phpstan this is implemented via bleeding-edge

Reply:
This is rather to keep changes as small as possible.